src/main/java/de/medizininformatik_initiative/process/data_sharing/DataSharingProcessPluginDefinition.java

```diff
@@ -83,11 +86,12 @@ public Map<String, List<String>> getFhirResourcesByProcessId()
 
 		return Map.of( //
 				ConstantsDataSharing.PROCESS_NAME_FULL_COORDINATE_DATA_SHARING, //
-				Arrays.asList(aCoo, cDaSh, nPrId, sTcoo, sTsenMer, sTsenRecHrp, tCoo, vDaSh), //
+				Arrays.asList(aCoo, cDaSh, nPrId, qReCon, sTcoo, sTsenMer, sTsenRecHrp, tCoo, vDaSh), //
```

Comment:
```suggestion
				List.of(aCoo, cDaSh, nPrId, qReCon, sTcoo, sTsenMer, sTsenRecHrp, tCoo, vDaSh), //
```
List.of(...) creates an immutable list which should be preferred here and at a few other locations in this repo.

Reply:
resolved with 6474a08